install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -431,12 +431,12 @@ data:
 {{- if .Values.bpf.natMax }}
   # bpf-nat-global-max specified the maximum number of entries in the
   # BPF NAT table.
-  bpf-nat-global-max: "{{ .Values.bpf.natMax | int }}"
+  bpf-nat-global-max: "{{ .Values.bpf.natMax | int | quote }}"
```

Comment:
On closer inspection, the use of `quote` here is wrong. Either use `foo: "{{ .Values.bar | int}}"`  or use `foo: {{ .Values.bar | int | quote}}` - but not both.

Reply:
Ah, I didn't notice that, thanks!

I fixed the above lines to make it consistent with other part of the code.

Fixed in https://github.com/cilium/cilium/pull/37670/commits/4708b740f6f60f2671187d1b304f3dd5ba07787d